test/hocr-wordfreq/hocr-wordfreq.tsht

```diff
@@ -4,10 +4,10 @@ TESTDATA="../testdata"
 plan 4
 
 normal=$(hocr-wordfreq $TESTDATA/sample.html | head -n 1)
-casesensitive=$(hocr-wordfreq -i $TESTDATA/sample.html | head -n 1)
+caseinsensitive=$(hocr-wordfreq -i $TESTDATA/sample.html | head -n 1)
```

Comment:
No, actually I would like to have that the default option is ignoring the cases and the special option `-i` will be case-sensitive. I assume that the second case is much less frequent in practice and therefore the first case should be default without any special options.